core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.ts

```diff
@@ -319,6 +390,44 @@ export class NgbdModalWorkflowExecutionsComponent implements OnInit {
     return "rgba(" + r + "," + g + "," + b + ",0.8)";
   }
 
+  updateCheckedSet(index: number, checked: boolean): void {
```

Comment:
please follow this PR for the checkbox logic https://github.com/Texera/texera/pull/1629/files